### PR TITLE
feat: vmind package

### PR DESCRIPTION
**Pull request by da730**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vmind/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (kkxxkk2019) [inline packages/vmind/index.html:6]: 这个文件的位置是不是不对

File: packages/vmind/src/chart-generation/data/challenge-词云.csv
Comment (kkxxkk2019) [inline packages/vmind/src/chart-generation/data/challenge-词云.csv:1]: 为啥用中文名称的文件

File: packages/vmind/yarn.lock
Comment (kkxxkk2019) [inline packages/vmind/yarn.lock:1]: 咋又 yarn.lock

Comment (da730) [reply packages/vmind/yarn.lock:1]: 忘记删了……

File: packages/vmind/src/chart-generation/data/challenge-词云.csv
Comment (da730) [reply packages/vmind/src/chart-generation/data/challenge-词云.csv:1]: 已移除

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**